Project.toml

```
@@ -1,7 +1,7 @@
 name = "Oceananigans"
 uuid = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
 authors = ["Climate Modeling Alliance and contributors"]
-version = "0.76.7"
+version = "0.77.1"
```

```suggestion
version = "0.77.0"
```

We haven't registered 0.77.0 yet so I think we either keep 0.77.0 and register it after merging. Or register 0.77.0 now (already on main, before merging this PR) and then 0.77.1 after merging this PR. (Did I make sense?)

---

How you prefer, we can register 0.77.0 now and 0.77.1 after merging this, but I would need it registered by tomorrow anyways. I ll register 0.77.0 now

---

Whatever works best. I don't have a preference

---

Thinking about it, since the changes are only in particle tracking, it's ok to register 0.77.0 after merging this PR. My rational for changing it to 0.77.1 was that I though 0.77.0 was already registered and I needed this code.